### PR TITLE
Imple alpine

### PR DESCRIPTION
**Pull request by ambareesha7**
I'm using alpinejs on front end for little functionalities like switching between tab's, intention of this is to minimise the calls to backend for trivial functions, as suggested by Sam 
but its still  work in progress

### CODE REVIEW

File: lib/archethic_web/live/transaction_details_live.ex [hunk 1]
```diff
@@ -19,9 +19,7 @@ defmodule ArchEthicWeb.TransactionDetailsLive do
        exists: false,
        previous_address: nil,
        transaction: nil,
-       hide_content: true,
        tab_panel: "tx",
```
Comment (samuel-uniris) [inline lib/archethic_web/live/transaction_details_live.ex:22]: I think you can clean the `tab_panel` and the `operation_section`

File: lib/archethic_web/templates/explorer/transaction_details.html.leex [hunk 2]
```diff
@@ -30,22 +28,30 @@
                         <% end %>
                     </div>
                 </div>
-
             <% end %>
-
-            <div class="is-2">
+            <div class="is-2" x-data="{
+                                tab_panel: '<%= @tab_panel %>',
```
Comment (samuel-uniris) [inline lib/archethic_web/templates/explorer/transaction_details.html.leex:33]: I think you can default as `tx` panel

File: transaction_HTTP_response.txt [hunk 3]
```diff
@@ -0,0 +1,130 @@
+START
```
Comment (samuel-uniris) [inline transaction_HTTP_response.txt:1]: This file should be removed also

**Review by samuel-uniris — CHANGES_REQUESTED** [review]

File: lib/archethic_web/live/transaction_details_live.ex [hunk 4]
```diff
@@ -72,7 +70,7 @@ defmodule ArchEthicWeb.TransactionDetailsLive do
 
   defp handle_transaction(
          socket,
-         tx = %Transaction{address: address, data: %TransactionData{content: content}}
+         tx = %Transaction{address: address, data: %TransactionData{content: _content}}
```
Comment (samuel-uniris) [inline lib/archethic_web/live/transaction_details_live.ex:73]: I think you can also clean this one

File: lib/archethic_web/live/transaction_details_live.ex (hunk 1 above)
Comment (samuel-uniris) [reply lib/archethic_web/live/transaction_details_live.ex]: Why do you need to keep this from the server side ?

File: lib/archethic_web/live/transaction_details_live.ex (hunk 4 above)
Comment (ambareesha7) [reply lib/archethic_web/live/transaction_details_live.ex:73]: transaction.data.content we are using it in leex file
Screenshot 2021-09-20 at 2 37 02 PM

Comment (samuel-uniris) [reply lib/archethic_web/live/transaction_details_live.ex:73]: We are already passing the transaction to the view, but here the pattern matching was used to determine if you show the content depending on its size, for performance issue.
So if you want to use this var, you should set this variable into the alpine.js data also.

Comment (ambareesha7) [reply lib/archethic_web/live/transaction_details_live.ex:73]: ok, i have removed "data: %TransactionData{content: _content"

**Review by samuel-uniris — APPROVED** [review]

**Comment by samuel-uniris** [comment]
Look good to me

**Pull request merged**